README.md

```diff
@@ -0,0 +1,166 @@
+Patreon Wordpress is a free plugin built and maintained by Patreon. When a patron pledges, they can then head over to www.yourwordpresssite.com and click your “Connect with Patreon” button to let the site know they are a patron. When you create content on WordPress you will be given the option to limit it to only paying patrons.
+
+This plugin is maintained by Patreon and will cover the basic use cases of patronage on WordPress. Looking for additional features? Extend with premium inter-operable WordPress plugins that others have developed, also found in this directory.
+
+### FEATURES
+
+- Keep your patrons engaged on your website
+- Convert your fans to patrons by showing them how much content they’d get
+- Publish patron-only posts with the full power of WordPress
+- Option to provide an ad-free experience to patrons as an added value
+
+### Installing the Patreon WordPress Plugin
+
+1. Install the plugin
+3. Activate the plugin and then click on ‘Patreon Settings’ to view the options page. It will look something like the below, copy the ‘Redirect URI‘ into the clip board and keep for the next step.
+![](https://c5.patreon.com/external/platform/wordpress-install-ss1.jpeg)
+
+### Creating an oAuth client on Patreon
+
+1. Ensure you are logged into Patreon, using your creator account. When it comes to testing this out you will want to have two Patreon accounts, one that acts as a patron and one being the content creator. The oAuth client needs to be created on your content creating account.
+2. Visit the [oAuth client page here](https://www.patreon.com/portal/registration/register-clients) and click "Create Client", fill out the form and add in an image URL for the icon that will appear to users when they are connecting to your website.
+3. Note the field for ‘Redirect URI‘, this should still be in your clipboard from the previous steps – if not copy/paste it exactly as it appears on the options page.
+4. Click the ‘Create Client’ button, and you should now see the new client in the list.
```

Comment:
here you do like weird unicode quotes, above you use double quotes. Be consistent and I might suggest using backticks for the cool `code` effect